README.md

```diff
@@ -12,13 +12,14 @@ pip install -r requirements.txt
 ```
 
 #### Usage and examples
+
 ```bash
-./pshtt_cli [options] DOMAIN...
-./pshtt_cli [options] INPUT
+python -m pshtt.cli [options] DOMAIN...
+python -m pshtt.cli [options] INPUT
```

Comment:
@garrettr Would it be easier to instruct people running it locally to install in developer mode with `pip install -e` and then just run `pshtt [options] DOMAIN...` instead?